### DISCUSSION TITLE
Backing up docker volumes

### DISCUSSION THREAD

**Discussion by SohaibBazaz** [outline/outline#7189] [Self Hosting]
Hey there, I have self-hosted Outline on Coolify using docker. I'm currently backing up all my services but couldn't find the container running Outline nor the volumes. I'm new to this so I'd like some guidance

**Comment by SohaibBazaz** [comment]
I ended up finding the volumes and successfully backing them up

Outline for some reason, sometimes, does NOT name it's volumes uniquely. In my case it the volume name was simply called "Storage-base". After figuring that out, I was able to back it up successfully

**Reply by tommoor** [reply]
Glad you figured it out, outline doesn't know anything about the volumes or their names - this is a docker concern.
